app/flask_config.py

```diff
@@ -5,6 +5,11 @@
 class Config(object):
     DEBUG = False
     TESTING = False
+    BERT = "https://tfhub.dev/google/bert_uncased_L-12_H-768_A-12/1"
+    MODEL = "UPR_2percent_ps0"
+    INSTANCE = "1573031002"
+    PATH_TO_GCS_CREDS = "./.credz/BERT Classification-9a8b5ef88627.json"
+    MODEL_CONFIG_PATH = "./static/model_config.json"
```

Comment:
Why is BERT, MODEL,INSTANCE not in there?

Reply:
Why *is* it or why is it *not* in there?

Reply:
This file is gone now. Obsolete comment